test/fuzz/tests/rpc_jsonrpc_server_test.go

```diff
@@ -0,0 +1,69 @@
+//go:build gofuzz || go1.18
+
+package tests
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/tendermint/tendermint/libs/log"
+	rs "github.com/tendermint/tendermint/rpc/jsonrpc/server"
```

Comment:
```suggestion
	rpcserver "github.com/tendermint/tendermint/rpc/jsonrpc/server"
```

Just so the name is more self-explanatory in context.